model/patch/patch.go

```diff
@@ -326,5 +327,9 @@ func (p *Patch) RemoveModulePatch(moduleName string) error {
 }
 
 func (p *Patch) IsGithubPRPatch() bool {
-	return p.GithubPatchData.PRNumber != 0
+	return p.GithubPatchData.BaseOwner != ""
```

Comment:
The get_project command pulls the patch from the database and checks out the correct commit based on information in the GitHubPatchData struct, which is now being multiplexed to give information for PR patch testing and merge testing. 
To tell the difference: we need the MergeCommitSHA for merge testing, but we don't need it for PR patches. The opposite is true for BaseOwner.